packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationAction.tsx

```diff
@@ -117,9 +117,9 @@ const CustomCalculationAction = ({
         }
         returnFn={async () => returnFn(false)}
       />
-      <div className='custom-calculation-action-container'>
-        <Fieldset legend='Custom Calculation Details' className='details-form'>
-          <Small className='field-legend'>
+      <div className='gmw-custom-calculation-action-container'>
+        <Fieldset legend='Custom Calculation Details' className='gmw-details-form'>
+          <Small className='gmw-field-legend'>
```

Comment:
No style exists for class `gmw-field-lengend` in any of the imported scss files.

Reply:
Actually, there are no styles for `gmw-field-legend` in any of the scss files. There was a style for `field-legend`, but you removed it. Is the `className` even needed here anymore? Did the UI change? Please provide before/after screenshots.

Reply:
Styling added back, bad deletion, thanks for the catch

Reply:
Done